tsconfig.json

```diff
@@ -0,0 +1,78 @@
+{
+    "compilerOptions": {
+        /* Basic Options */
+        "target":
+            "ES3" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */,
```

Comment:
Es3 ?

Reply:
Isn't it `tsc` which is used to build now according to the package.json files ?

Reply:
yes indeed